apps/client/src/styles/theme.scss

```diff
@@ -213,6 +213,14 @@ $_tertiary: map.merge(map.get($_palettes, tertiary), $_rest);
     );
   }
 
+  @include mat.dialog-overrides(
+    (
+      container-max-width: 80vw,
+      container-shape: 4px,
```

Comment:
I think the new radius is fine, only on mobile (full-screen) it should be tiny.

What about moving `container-shape: 4px` into `@media (max-width: 575.98px) {}`?